airflow/bin/cli.py

```diff
@@ -707,6 +707,9 @@ class CLIFactory(object):
         'conf': Arg(
             ('-c', '--conf'),
             "json string that gets pickled into the DagRun's conf attribute"),
+        'date': Arg(
```

Comment:
Rather than adding this to the args for trigger_dag, can you just use the chared arg `execution_date`, above? (line 625)